### PR TITLE
layers: Add NotTestable as a valid testname entry

### PR DESCRIPTION
**Pull request by daveh-lunarg**
Modify spec.py to add the keyword NotTestable as a valid test name.
Use for VU checks that are implemented but which have no reasonable
method of testing.

### CODE REVIEW

File: layers/spec.py [hunk 1]
```diff
@@ -256,7 +256,7 @@ def genDB(self, db_file):
         db_lines.append("# <error_enum>%s<check_implemented>%s<testname>%s<api>%s<errormsg>%s<note>" % (self.delimiter, self.delimiter, self.delimiter, self.delimiter, self.delimiter))
         db_lines.append("# error_enum: Unique error enum for this check of format %s<uniqueid>" % validation_error_enum_name)
         db_lines.append("# check_implemented: 'Y' if check has been implemented in layers, or 'N' for not implemented")
-        db_lines.append("# testname: Name of validation test for this check, 'Unknown' for unknown, or 'None' if not implmented")
+        db_lines.append("# testname: Name of validation test for this check, 'Unknown' for unknown, 'None' if not implmented, or 'NotTestable' if cannot be implmented")
```
Comment (chrisforbes) [inline layers/spec.py]: implemented (presumably copy/paste from existing error in this line)

**Comment by chrisforbes** [comment]
I'm curious which checks you plan to make NotTestable?

File: layers/spec.py (hunk 1 above)
Comment (daveh-lunarg) [inline layers/spec.py]: I'll implment the spelling correction.

e.g., VU 01217, 'extent.depth must be a multiple of the compressed texel block depth'. All supported compressed formats have block depth of 1, so no test can currently be written to 
fail this VU.

Comment (daveh-lunarg) [inline layers/spec.py]: Why can't I reply to the other comment?

In any case, an example of NotTestable:
VU 01217, 'extent.depth must be a multiple of the compressed texel block depth'. All supported compressed formats have block depth of 1, so no test can currently be written to 
fail this VU.

**Pull request merged**